tests/TestConfig.conf

```diff
@@ -0,0 +1,17 @@
+# This is an example config file, you can adapt it to your needs.
+# All text following a # character is treated as comments
+
+[RunControl]
+RunSizeLimit = 1000000000
+
+[DataCollector]
+FilePattern = "../data/run$6R$X"
+
+[LogCollector]
+SaveLevel = EXTRA
+PrintLevel = INFO
+
+[Producer.TLU]
+ConfParameter = 123
```

Comment:
Actually I dont know, but this is not used, can be probably removed. I copied this from bdaq... This is just a stupid config file in order to test the producer with EUDAQ.

Reply:
That's just for testing to see if one can get a config setting via EUDAQ. I added this feature to [pyEUDAQwrapper](https://github.com/eudaq/eudaq/pull/470), but we do not use it, yet?

Reply:
In case of pytlu `DutMask`, `AndMask` and `TriggerInterval` are read from an EUDAQ config file. All other parameters are ignored (do not exist in pytlu).